tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -1861,12 +1861,13 @@ void CorProfiler::InitializeTraceMethods(WCHAR* id, WCHAR* integration_assembly_
 
 void CorProfiler::InstrumentProbes(debugger::DebuggerMethodProbeDefinition* methodProbes, int methodProbesLength,
```

Comment:
To solve the PInvoke issue, you may want to keep the original InstrumentProbes and add this second overload one

Reply:
https://github.com/DataDog/dd-trace-dotnet/pull/3955#discussion_r1152527650